### ISSUE TITLE
encoder configuration incorrect for certain cases

### ISSUE THREAD

**Issue by CyrilZwahlen** [Opendigitalradio/ODR-AudioEnc#3]
if only one channel is being used, the encoder configuration is set wrong for certain bitrates.
suggestion - update odr-audioenc.cpp as follows:

replace
```
    if (*aot == AOT_NONE) {

        if(channels == 2 && subchannel_index <= 6) {
            *aot = AOT_DABPLUS_PS;
        }
        else if((channels == 1 && subchannel_index <= 8) || subchannel_index <= 10) {
            *aot = AOT_DABPLUS_SBR;
        }
        else {
            *aot = AOT_DABPLUS_AAC_LC;
        }
    }
```
with
```
    if (*aot == AOT_NONE) {

        if(channels == 2 && subchannel_index <= 6) {
            *aot = AOT_DABPLUS_PS;
        }
        else if((channels == 1 && subchannel_index <= 8) || (channels == 2 && subchannel_index <= 10)) {
            *aot = AOT_DABPLUS_SBR;
        }
        else {
            *aot = AOT_DABPLUS_AAC_LC;
        }
    }
```

**Comment by mpbraendli** [comment]
Yep, good observation. Thank you!

**Comment by mpbraendli** [comment]
Fixed in b68380ba3c00f6db0d60ec5fe9ec72bb230ea7ad
Thanks for the report!
